src/main/java/com/greghaskins/spectrum/Spectrum.java

```diff
@@ -250,22 +250,24 @@ public static void beforeEach(final com.greghaskins.spectrum.Block block) {
   }
 
   /**
-   * Declare a {@link Block} to be run after each spec in the current suite.
+   * Declare a {@link com.greghaskins.spectrum.Block Block} to be run
+   * after each spec in the current suite.
```

Comment:
The existing hyperlink referenced a ~deprecated~ class: I merely updated the reference to the class that is not deprecated.